sql/server_component/clone_protocol_service.cc

```diff
@@ -539,6 +545,13 @@ DEFINE_METHOD(int, mysql_clone_get_response,
   *net_length = 0;
   *length = my_net_read(net);
 
+  /** The ZSTD object has been updated to include the newly created decompressor
+   * context. Include the change in the net extension so that the resource can
```

Comment:
This and the next line: the comment style with leading stars is not used in MySQL:
```suggestion
   context. Include the change in the net extension so that the resource can
```